### PR TITLE
Cleanup garbage collection test for container type

### LINKED ISSUE #118: Adjust location of unit tests to suit the scope of responsibility
Currently, Yorkie's test coverage is relatively low at 56%. This is due to the go cover's coverage measurement method. The default measurement method is for each test to analyze only the package being tested(https://golang.org/cmd/go/#hdr-Testing_flags, Using `-coverpkg` applies coverage analysis in each test to packages matching the patterns). Some packages are being tested outside.

So @dc7303 suggested increasing coverage by adjusting the location of unit tests to suit the scope of responsibility of the package.

### PR DESCRIPTION
**Pull request by dc7303**
**What this PR does / why we need it**:
Organized so that garbage collection tests for container types can be
managed in a location that fits the scope of responsibility.

**Which issue(s) this PR fixes**:

Address #118 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation**:

```docs

```

**Checklist**:
- [x] Added relevant tests or not required
- [x] Didn't break anything

### CODE REVIEW

File: pkg/document/json/root_test.go [hunk 1]
```diff
@@ -127,4 +127,39 @@ func TestRoot(t *testing.T) {
...
 		assert.Equal(t, 1, nodeLen)
 	})
+
+	t.Run("garbage collection for container test", func(t *testing.T) {
+		root := testhelper.TestRoot()
+		ctx := testhelper.TextChangeContext(root)
+
+		newInt := func(n int) *json.Primitive {
+			return json.NewPrimitive(n, ctx.IssueTimeTicket())
+		}
```
Comment (hackerwins) [inline pkg/document/json/root_test.go:137]: I think the benefit of function extraction is small. This function feels like an early abstraction. 🤔
What if we put it back inline?

**Review by hackerwins — APPROVED** [review]
Thank you for your contribution. I left a minor comment, so I change this to approval.

File: pkg/document/json/root_test.go (hunk 1 above)
Comment (dc7303) [reply pkg/document/json/root_test.go]: Thanks for the feedback! :D Your requested changes have been reflected.

**Comment by hackerwins** [comment]
Could you fix conflicting files?

**Pull request merged**